javascript/funWithFunctions.md

```diff
@@ -0,0 +1,28 @@
+## Functional Expressions in D&D ##
+I've been having fun with functions in creating a JavaScript version of the 5th edition D&D character sheet.
+
+In recreating the character bonus table I got thru 47 lines of inline conditionals before the formula slapped me in the face:  
+![](http://cl.ly/image/3046350l3X1r/OG_s.jpg)
+
+
+Even in adding abilities to an array, I was still left with 6 variables and a declared function:
```

Comment:
This sentence reads a little oddly. Perhaps "Even after I used an array to specify the abilities of the character, I ..." or something?